AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/Model/KeyStoreAdmin.smithy

```diff
@@ -295,7 +307,20 @@ structure InitializeMutationInput {
   @documentation("Optional. Defaults to TrustStorage. See System Key.")
   SystemKey: SystemKey
 
-  @documentation("Optional. Defaults to False. As of v1.9.0, setting this true throws a UnsupportedFeatureException.")
+  @documentation(
+  "Optional. Defaults to False, which Versions (or Rotates) the Branch Key,
+  creating a new Version that has only ever been in the terminal state.
+  Setting this value to True disables the rotation.
+  This is a Security vs Performance trade off.
+  Mutating a Branch Key can change the security domain of the Branch Key.
+  Some application's Threat Models benefit from ensuring a new Version
+  is created whenever a Mutation occurs,
+  allowing the application to track under which security domain data
+  was protected.
+  However, not all Threat Models call for this.
+  Particularly if a Mutations are triggered in response to external actors,
+  creating a new Version for every Mutation request can needlessly grow
+  the item count of a Branch Key.")
```

Comment:
```suggestion
  However, not all Threat Models call for this.
  Particularly if a Mutations are triggered in response to external actors,
  creating a new Version for every Mutation request can needlessly grow
  the item count of a Branch Key.")
```
```suggestion
  However; not all Threat Models call for this.
  Particularly if a Mutation is triggered in response to external actors,
  creating a new Version for every Mutation request can needlessly grow
  the item count of a Branch Key.")
```

Reply:
I am not sure I agree with the substitution of `;` for a comma `,`.

I will consult my style guides...

Reply:
I think the greater English sin is starting a sentence with `However`;
which is a conjunction.
On this occasion, I would like to commit that English sin;
I shall do my penance at a later date.

I will fix the pluralization issue.